hs/src/Reach/Verify/SMT.hs

```diff
@@ -1132,6 +1132,9 @@ smt_e at_dv mdv de = do
       mempty
     DLE_TokenInit {} ->
       mempty
+    DLE_CanReceive at _ _ ->
+      -- TODO - I'm really uncertain what to do here, this just lets me get past this phase.
+      unbound at
```

Comment:
This is fine. It says that the verifier can't guess whether it is accepted or not.